hw/ip/rstmgr/data/rstmgr_sec_cm_testplan.hjson

```diff
@@ -25,27 +25,52 @@
   testpoints: [
     {
       name: sec_cm_bus_integrity
-      desc: "Verify the countermeasure(s) BUS.INTEGRITY."
+      desc: '''Verify the countermeasure(s) BUS.INTEGRITY.
+            This entry is covered by tl_access_test.
+            '''
       milestone: V2S
       tests: ["rstmgr_tl_intg_err"]
     }
     {
       name: sec_cm_scan_intersig_mubi
-      desc: "Verify the countermeasure(s) SCAN.INTERSIG.MUBI."
+      desc: '''Verify the countermeasure(s) SCAN.INTERSIG.MUBI.
+
+            **Stimulus**:
+            Same as smoke test but drive scanmode_i with a constant invalid
+            value during the test.
+
+            **Check**:
+            If dut accepts any of invalid values, test will fail by turing dut to scanmode.
+            '''
       milestone: V2S
       tests: ["rstmgr_sec_cm_scan_intersig_mubi"]
     }
     {
       name: sec_cm_leaf_rst_bkgn_chk
-      desc: "Verify the countermeasure(s) LEAF.RST.BKGN_CHK."
+      desc: '''Verify the countermeasure(s) LEAF.RST.BKGN_CHK.
+
+            ** Stimulus**:
+            Execute a series of reset event - lowpower, hwreq, ndm, and
+            sw reset -. And at the beginning of these events, create
+            reset consistency error to one of 27 leaf modules.
+            Do the same test for all 27 modules.
+
+            **Check**:
+            Upon asserting each reset consistency error,
+            check alert_fatal_cnsty_fault is asserted.
+            '''
       milestone: V2S
-      tests: []
+      tests: ["rstmgr_leaf_rst_cnsty"]
     }
     {
       name: sec_cm_leaf_rst_shadow
-      desc: "Verify the countermeasure(s) LEAF.RST.SHADOW."
+      desc: '''Verify the countermeasure(s) LEAF.RST.SHADOW.
+            After power up, create glitch to a shadow leaf reset module.
```

Comment:
do we also want to check the case where normal is glitched but shadow is not? both are possible.

Reply:
good point. I will update sequence to do that.

Reply:
update test in #11909